src/VisualStudio/CSharp/Test/PersistentStorage/CloudCachePersistentStorageTests.cs

```diff
@@ -86,6 +83,9 @@ private static CacheContainerKey Convert(CloudCacheContainerKey containerKey)
             public void Dispose()
                 => (_cacheService as IDisposable)?.Dispose();
 
+            public ValueTask DisposeAsync()
+                => (_cacheService as IAsyncDisposable)?.DisposeAsync() ?? ValueTaskFactory.CompletedTask;
```

Comment:
`ValueTaskFactory.CompletedTask`
You know that `default` works here, right?
You need `Task.CompletedTask` because `Task` is a reference type. But as a value type `ValueTask` was written to behave as you might expect in its default value.

Reply:
`default` works for `ValueTask`, but it doesn't work for `ImmutableArray<T>`. `ValueTaskFactory.CompletedTask` has the advantage of not needing to know which kind of value type this is.

Reply:
I'm with sam on this.   I don't like 'default' here as it's vague and confusing.  `CompletedTask` makes it abundantly clear what it means.